virt/virt-2-4-release-notes.adoc

```diff
@@ -54,6 +54,9 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 == Notable technical changes
 //Placeholder for new content.
 
+//CNV-5465-Disconnected installation
+* {VirtProductName} can be installed on disconnected clusters in restricted network environments that do not have Internet connectivity. You can create local mirrors for the {VirtProductName} operator and install from a local registry that is accessible to the disconnected cluster.
```

Comment:
`from a local registry` -> `from a local Operator catalog image`

Maybe we can also add (updating it for 4.5) for 4.5 a link to the relevant page on OCP doc:
https://docs.openshift.com/container-platform/4.4/operators/olm-restricted-networks.html

Reply:
@tiraboschi I will check with my team whether a link from the release notes to the product documentation is possible. If it is, I will look into adding a link.

Reply:
@stu-gott @rmohr @tiraboschi - Please advise if your review of the PR is complete, and if it can  move to the QE and Peer Review stages? In doc, we need to complete all PRs (not the release note PRs, but regular release PRs) this week, therefore if you are satisfied, please approve this PR for the next review stage and I will move it forward. Thanks!

Reply:
I see that the link was updated.

@lmandavi, @tiraboschi also made reference to changing the wording of "registry". Does this need to be changed? Please consider my opinion neutral on the matter -- I'm just trying to make sure all concerns are addressed.

Reply:
@stu-gott @tiraboschi Yes, it has been changed : from a local registry -> from a local Operator catalog image.